Logos/cisco-logo-72px.svg

```diff
@@ -1,20 +1,21 @@
-            <path d="M3,12.4716981 C3,11.6792453 2.375,11 1.5,11 C0.625,11 0,11.6792453 0,12.4716981 L0,15.5283019 C0,16.3207547 0.75,17 1.5,17 C2.375,17 3,16.3207547 3,15.5283019 L3,12.4716981" id="48892c1b-839d-4bc5-91b4-c663ca4054dd"/>
-            <path d="M12,7.63829787 C12,6.81914894 11.25,6 10.5,6 C9.625,6 9,6.70212766 9,7.63829787 L9,15.3617021 C9,16.2978723 9.75,17 10.5,17 C11.375,17 12,16.2978723 12,15.3617021 L12,7.63829787" id="611430d8-9693-41de-99c7-2f35c85eec82"/>
-            <path d="M21,1.85483871 C21,0.865591398 20.25,0 19.5,0 C18.625,0 18,0.865591398 18,1.85483871 L18,21.1451613 C18,22.1344086 18.75,23 19.5,23 C20.375,23 21,22.1344086 21,21.1451613 L21,1.85483871" id="852c186e-ce2d-4152-adce-9f838311fd4d"/>
-            <path d="M30,7.63829787 C30,6.81914894 29.25,6 28.5,6 C27.625,6 27,6.70212766 27,7.63829787 L27,15.3617021 C27,16.2978723 27.75,17 28.5,17 C29.375,17 30,16.2978723 30,15.3617021 L30,7.63829787" id="b99c66ef-3754-4b2e-880b-080b9c96855f"/>
-            <path d="M38,12.4716981 C38,11.6792453 37.25,11 36.5,11 C35.625,11 35,11.6792453 35,12.4716981 L35,15.5283019 C35,16.3207547 35.75,17 36.5,17 C37.375,17 38,16.3207547 38,15.5283019 L38,12.4716981" id="1cc745b9-c34a-4459-8d09-67e1f0a7e4c3"/>
-            <path d="M47,7.63829787 C47,6.81914894 46.25,6 45.5,6 C44.625,6 44,6.70212766 44,7.63829787 L44,15.3617021 C44,16.2978723 44.625,17 45.5,17 C46.375,17 47,16.2978723 47,15.3617021 L47,7.63829787" id="eee67ecc-f43c-4348-a1c0-d20d3d15934b"/>
-            <path d="M56,1.85483871 C56,0.865591398 55.375,0 54.5,0 C53.625,0 53,0.865591398 53,1.85483871 L53,21.1451613 C53,22.1344086 53.75,23 54.5,23 C55.375,23 56,22.1344086 56,21.1451613 L56,1.85483871" id="0215bbe7-b01e-4b17-97b7-d4eaec1a1228"/>
-            <path d="M64,7.63829787 C64,6.81914894 63.375,6 62.5,6 C61.625,6 61,6.70212766 61,7.63829787 L61,15.3617021 C61,16.2978723 61.625,17 62.5,17 C63.375,17 64,16.2978723 64,15.3617021 L64,7.63829787" id="d5468f7e-db62-42a1-9c6d-d0e2193209e0"/>
-            <path d="M72,12.4716981 C72,11.6792453 71.25,11 70.5,11 C69.625,11 69,11.6792453 69,12.4716981 L69,15.5283019 C69,16.3207547 69.625,17 70.5,17 C71.375,17 72,16.3207547 72,15.5283019 L72,12.4716981" id="7a0dcd37-2cc9-4d43-95f0-28ccd746441b"/>
-        </g>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72">
```

Comment:
Please add xmlns="http://www.w3.org/2000/svg" in svg tag.

Thanks,
Aditi Maheshwari